taprpc/taprootassets.proto

```diff
@@ -693,7 +697,8 @@ message DecodedProof {
     // The index depth of the decoded proof, with 0 being the latest proof.
     uint32 proof_at_depth = 1;
 
-    // The total number of proofs contained in the raw proof.
+    // The total number of proofs contained in the raw proof (this will always
+    // be 1 if a single mint/transition proof was given as the raw_proof).
     uint32 number_of_proofs = 2;
```

Comment:
> this will always be 1 if a single mint/transition proof was given as the raw_proof

In the comment above you say that `raw_proof` must be a file. In which case, it can not consist of 1 transition proof.

I think you should make this change in the highlighted comment:
`a single mint/transition proof was given as the raw_proof` -> `the serialised proof file given in raw_proof contained a single mint (issuance) proof`

Reply:
In this case we actually explicitly allow either a single proof or a proof file (which _could_ just contain a single proof, but would look differently when encoded as a file) to be given. But yeah, maybe my phrasing wasn't optimal. How about the new version?

Reply:
Got it, looks good.